docs/schema_additions.md

```diff
@@ -29,6 +30,11 @@ elaborated in the next section. All strings contain UTF-8 text.
 {
     /* ... other OSV fields ... */
     "database_specific": {
+		"iocs": {
```

Comment:
nit: is the indentation inconsistent here with line 38?

Reply:
Fixed.